components/serializer.rst

```diff
@@ -755,7 +755,7 @@ Here, we set it to 2 for the ``$child`` property:
 The metadata loader corresponding to the chosen format must be configured in
 order to use this feature. It is done automatically when using the Symfony
 Standard Edition. When using the standalone component, refer to
-:ref:`the groups documentation <component-serializer-attributes-groups>` to
+:ref:`the groups documentation <attributes-groups>` to
```

Comment:
Sorry? The current link goes to `http://symfony.com/doc/master/components/serializer.html#component-serializer-attributes-groups` which doesn't exist, but `http://symfony.com/doc/master/components/serializer.html#attributes-groups` does and thus why I changed it to  `attributes-groups`. I've verified this on the version 4 of the docs on the website.